cysync/src/errors/error.ts

```diff
@@ -33,6 +33,15 @@ class DisplayError {
   public getCode() {
     return this.code || '';
   }
+  /**
+   * Errors are structure in a way that the second sentence of the error
+   * message can be substituted for the action message.
+   * Mostly needed on Tooltips
+   * @returns action message
+   */
+  public getActionMessage() {
+    return this.message.split('.')[1];
+  }
```

Comment:
Shouldn't we have a length check here? And also a warning log and default action message if the length is not correct?

Reply:
May I know why do you feel we would need a length check here?

Reply:
1. Need a default action message
2. length check after splitting
3. log if there is no second sentence